src/Symfony/Component/Process/Process.php

```diff
@@ -1198,6 +1214,30 @@ public function setEnhanceSigchildCompatibility($enhance)
     }
 
     /**
+     * Sets whether environment variables will be inherited or not.
+     *
+     * @param bool $inheritEnv
+     *
+     * @return ProcessBuilder
+     */
+    public function inheritEnvironmentVariables($inheritEnv = true)
```

Comment:
I'd personally prefer `inheritEnv()` here, but I borrowed this name from ProcessBuilder, consistency first :)